runtime/commonMain/src/kotlinx/benchmark/ReportBenchmarksStatistics.kt

```diff
@@ -86,13 +86,17 @@ class ReportBenchmarksStatistics(values: DoubleArray) {
     }
 }
 
-expect fun Double.format(precision: Int): String
+expect fun Double.format(precision: Int, useGrouping: Boolean = true): String
```

Comment:
Please add tests to `runtime/commonTest/src/kotlinx/benchmark/tests/FormatTests.kt` with `useGrouping = false`